### PR TITLE
fix(web): several UI fix 

### PR DESCRIPTION
**Pull request by airslice**
# Overview

- Fix range field input
- Fix assets accept type for sketch custom property fields
- Add placeholder to style node component expression input
- Remove translations for style node title
- Remove min width for popup menu TitleWrapper
- Fix property list custom DnD with handle
- Correct the `Delete` button text
- Fix the title missing in story block edit fields
- Add some missing translations
- Store the layout of assets on dashboard assets page instead of AssetsManager
- Store the layout of projects by workspace

## Release Notes

- **New Features**
  - Introduced a new `Assets` component for managing and displaying assets in the dashboard.
  - Added support for layout management in the `AssetsManager` component with new `layout` and `onLayoutChange` props.
  - Enhanced timeline playback functionality with a new `PlayButton` component and refined controls.
  - Added a new `linkButtonStoryBlock` for improved storytelling interactions.

- **Improvements**
  - Updated translation files for English and Japanese, expanding localization for various UI elements and messages.
  - Streamlined handling of asset types and input fields for better user experience.
  - Enhanced clarity and usability of field titles and descriptions across multiple components.

- **Bug Fixes**
  - Standardized title properties across multiple field types to improve consistency in the user interface.

- **Chores**
  - Refined internal logic for managing local storage keys for project views, ensuring workspace-specific layout preferences.

### CODE REVIEW

**Review by asahina820 — COMMENTED** [review]
I checked Japanese, LGTM!

File: web/src/beta/features/Dashboard/ContentsContainer/Assets/index.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import AssetsManager from "@reearth/beta/features/AssetsManager";
+import { ManagerLayout } from "@reearth/beta/ui/components/ManagerBase";
+import { FC, useCallback, useState } from "react";
+
+const ASSETS_LAYOUT_STORAGE_KEY = `reearth-visualizer-dashboard-assets-layout`;
+
+type Props = {
+  workspaceId?: string;
+};
```
[the remark this answers is not in the document]
Comment (airslice) [reply web/src/beta/features/Dashboard/ContentsContainer/Assets/index.tsx:9]: Updated

File: web/src/beta/features/Dashboard/ContentsContainer/Projects/hooks.ts [hunk 2]
```diff
@@ -199,19 +199,34 @@ export default (workspaceId?: string) => {
...
+
   const [layout, setLayout] = useState(
     ["grid", "list"].includes(
-      localStorage.getItem(PROJECTS_VIEW_STATE_STORAGE_KEY) ?? ""
+      localStorage.getItem(projectsViewStateStorageKey) ?? ""
     )
-      ? (localStorage.getItem(PROJECTS_VIEW_STATE_STORAGE_KEY) as ManagerLayout)
+      ? (localStorage.getItem(projectsViewStateStorageKey) as ManagerLayout)
       : "grid"
   );
```
[the remark this answers is not in the document]
Comment (airslice) [reply web/src/beta/features/Dashboard/ContentsContainer/Projects/hooks.ts]: Updated

File: web/src/beta/ui/fields/RangeField.tsx [hunk 3]
```diff
@@ -23,18 +25,52 @@ const RangeField: FC<RangeFieldProps> = ({
...
+    (value: number | undefined) => {
+      setInputValues((prev) => {
+        const next = [
+          prev[0],
+          value === undefined
+            ? max
+            : max !== undefined && value > max
+              ? max
+              : value
+        ];
```
[the remark this answers is not in the document]
Comment (airslice) [reply web/src/beta/ui/fields/RangeField.tsx:67]: Updated.

**Pull request merged**